proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -393,6 +393,12 @@ message GetChecksByFederatedGraphNameRequest {
 }
 
 message SchemaCheck {
+  message SchemaCheckGhDetails {
```

Comment:
You don't have to prefix it with `SchemaCheck` because it is a nested message and will be prefixed already.

Reply:
Removed prefix